src/nft/components/sell/select/SelectPage.css.ts

```diff
@@ -0,0 +1,66 @@
+import { style } from '@vanilla-extract/css'
+
+import { sprinkles, vars } from '../../../css/sprinkles.css'
+
+export const section = style([
+  sprinkles({
+    paddingLeft: { sm: '16', lg: '0' },
+    paddingRight: { sm: '16', lg: '0' },
+  }),
+  { maxWidth: '1000px', margin: '0 auto' },
+])
+
+export const sellAssetSelected = style({
+  border: `4px solid ${vars.color.genieBlue} !important`,
```

Comment:
seems best to avoid !Important, we may have a lint rule but think it's not enabled for sprinkles. are there alternatives in this case?